src/components/common/toast/Toast.tsx

```diff
@@ -0,0 +1,87 @@
+import { FC, useEffect, useRef, useState } from "react";
+import ToastBase from "@/components/common/toast/toast.svg";
+import { CVoxelThree } from "@/interfaces";
+import CVoxelPresenter from "@/components/CVoxel/CVoxelPresenter";
+import { Canvas, useFrame } from "@react-three/fiber";
+import { OrbitControls, PerspectiveCamera } from "@react-three/drei";
+import * as THREE from "three";
+import { useCVoxelToast } from "@/hooks/useCVoxelToast";
+import clsx from "clsx";
+
+const Presenter = ({ voxel }: { voxel: CVoxelThree }) => {
+  const cCollectionRef = useRef<THREE.Group>(new THREE.Group());
+
+  useFrame(() => {
+    cCollectionRef.current.rotation.y += 0.005;
+  });
+
+  return (
+    <>
+      <ambientLight intensity={1.0} />
+      <directionalLight
+        castShadow
+        position={[0, 8, 0]}
+        intensity={1}
+        shadow-mapSize-width={1024}
+        shadow-mapSize-height={1024}
+        shadow-camera-far={50}
+        shadow-camera-left={-10}
+        shadow-camera-right={10}
+        shadow-camera-top={10}
+        shadow-camera-bottom={-10}
+      />
+      <OrbitControls
+        enablePan={false}
+        enableZoom={false}
+        enableRotate={false}
+      />
+      <group ref={cCollectionRef} position={[0, -1.7, 0]}>
+        <CVoxelPresenter
+          {...{
+            ...voxel,
+            position: new THREE.Vector3(0, 0, 0),
+            offset: new THREE.Vector3(0, 0, 0),
+            scale: 1.0,
+          }}
+        />
+      </group>
+      <PerspectiveCamera makeDefault position={[10, 6, 10]} zoom={3} />
+    </>
+  );
+};
+
+export const Toast: FC<{}> = () => {
+  const { toast } = useCVoxelToast();
+  const [isMount, setIsMount] = useState<boolean>(false);
+
```

Comment:
How about that?
```
if(!toast.isShow){
 return <></> ( or, return null)
}

Reply:
I don’t try it yet so not sure that it works though.

Reply:
I thought it looked good too, but when I actually moved it, the component disappeared the moment it was closed, so the animation was not rendered.

Reply:
oh, I see. In that case that's okay. Thanks!